src/components/BookState.css

```diff
@@ -0,0 +1,111 @@
+.book-card {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 30px;
+  background-color: #fff;
+  padding: 20px;
+  border: solid 1px #eee;
+}
+
+.book-details {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-family: 'RobotoSlab', sans-serif;
+}
+
+.book-catagory {
+  color: #121212;
+  opacity: 0.5;
+  font-size: 14px;
+  margin-bottom: 10px;
+}
+
+.secondary-header {
+  color: #121212;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 22px;
+  margin-bottom: 5px;
+}
+
+.book-author {
+  color: #4386bf;
+  font-size: 14px;
+  margin-bottom: 8px;
+  font-family: 'RobotoSlab', sans-serif;
+}
```

Comment:
- [ ] As I can see, you are repeating the same rule again and again. (font-family). Please create one class for the elements who shares the same font family or you can just add this rule into your body selector to apply all elements.